src/App.tsx

```diff
@@ -1,10 +1,11 @@
-import "./App.css";
+import "./styles/App.css";
+import { NextUIProvider } from "@nextui-org/react";
 
 function App() {
   return (
-    <>
-      <h1 className="text-3xl font-bold underline">Whimsical Manga Reader</h1>
-    </>
+    <NextUIProvider>
+      <></>
+    </NextUIProvider>
```

Comment:
You can remove the fragment inside the `NextUIProvider` unless it is required.

Reply:
It is required, I just don't have anything inside because when I was doing the last commits I just cleared all my testing elements. But I do require it to use NextUI library